src/transaction-request.js

```diff
@@ -274,6 +274,17 @@ class TransactionRequest {
    * ];
    */
   deleteRows(table, keys, callback) {
+    if (!this.transaction) {
```

Comment:
@callmehiphop what do you think about this approach?

Reply:
Seems fine to me! I'm not sure if its within the scope of this PR but we could probably do away with the Table extending the TransactionRequest.